### PR TITLE
Premier success service and support offering

### PR DESCRIPTION
**Pull request by tamigoodall**

### CODE REVIEW

File: articles/services/premier-success.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+* Roll-up of all your details about use cases, subscription information, legacy type, contractual obligations, targeted go-live dates and license roll-out plans.
+* Program dates and project timelines
+* Identified outcomes and measurable goals
+* Knowledge transfer and learning opportunities 
+
+## Operations and Dashboard Review
+
+An Operations and Dashboard Review is provided monthly. The goal of this review is to better understand and align with you around issues, upcoming activities, Auth0 change management, and issue trending/analysis. Key topics include:
+
+* Support ticket review and analysis
```
Comment (imadni15) [inline articles/services/premier-success.md]: Lets remove this line on ticket reviews. In speaking with Dave Wilner he didn't want us to highlight ticket review

File: articles/support/index.md [hunk 2]
```diff
@@ -35,51 +37,61 @@ Auth0 offers the following support plans:
...
 Please contact the [Auth0 Sales Team](https://auth0.com/?contact=true) if you have specific support requirements or are interested in the Enterprise Support Plan (with or without the Preferred Support option).
 
-## Support Channels
+### Premier Success support
+
+You can add a Premier Success support option to your subscription. With that option, all your inquiries are funneled to a dedicated team of support engineers who understand your use cases, projects, and plans. Your Premier Support Team will be able to quickly investigate and ultimately resolve your issues sooner. Your support plan will also include: 
+
+* Weekly operations reviews with a dedicated Customer Success Manager
+* Issue tracking dashboard
+* 24/7/365 phone support
```
Comment (imadni15) [inline articles/support/index.md]: Lets just make this 24/7 as opposed to 24/7/365. I know Kenny and I had spoken on 24/7 but I am not sure he is committed to 24/7/365

File: articles/services/premier-success.md (hunk 1 above)
Comment (tamigoodall) [reply articles/services/premier-success.md]: Done

**Comment by tamigoodall** [comment]
Delayed until early August.

**Pull request closed without merging**